wger.py

```diff
@@ -14,24 +14,34 @@ def __repr__(self):
     def user_login(self):
         """Login Function"""
         login_url='https://wger.de/en/user/login'
-        user=requests.get(login_url)
 
-        xsrftoken = user.cookies['csrftoken']
-        session_id = user.cookies['sessionid']
+        user=requests.get(login_url)
+        xsrftoken = user.cookies.get('csrftoken')
+        session_id = user.cookies.get('sessionid')
         cookie_full=f"csrftoken={xsrftoken}; sessionid={session_id}"
 
-        header={'Authorization': self.auth_token,'X-CSRFToken':xsrftoken,'Cookie':cookie_full,'Referer':login_url}
+        header={'Authorization': self.auth_token,
+                'X-CSRFToken':xsrftoken,
+                'Cookie':cookie_full,
+                'Referer':login_url}
 
-        login_data= {'username': self.username, 'password': self.password, 'submit':'Login'}
+        login_data= {'username': self.username,
+                     'password': self.password,
+                     'submit':'Login'}
         req=requests.post(url=login_url, data=login_data, headers=header)
         return req.content
 
 
-    def get_info(self,url_file):
+    def get_info(self,url_file,offset=20):
         """General function for GET requests"""
         get_url=self.base_url+url_file
-        response = requests.get(url=get_url,headers={'Authorization': self.auth_token})
-        return response.json()
+        response=requests.get(url=get_url,headers={'Authorization': self.auth_token}).json()
```

Comment:
Verify the response code of your calls. What happens if the call fails and you try to transform it into JSON format?